client/state/rewind/selectors/get-backup-retention-update-status.ts

```diff
@@ -0,0 +1,13 @@
+import type { AppState } from 'calypso/types';
+
+/**
+ * Returns the update backup retention request's status.
+ *
+ * @param state The application state.
+ * @param siteId The site ID for which to retrieve days of backups saved.
+ * @returns The the status of the request.
+ */
+const getBackupRetentionUpdateRequestStatus = ( state: AppState, siteId: number ): string =>
+	state.rewind[ siteId ]?.retention?.updateBackupRetentionRequestStatus ?? 'unsubmitted';
```

Comment:
Let's make sure that we use optional chaining only when really necessary. For example, in this case, if `rewind[ siteId ]` exists, then its `retention` property will exist, as well as its `updateBackupRetentionRequestStatus` sub-property.